icu4j/main/classes/core/src/com/ibm/icu/message2/PlainStringFormattedValue.java

```diff
@@ -0,0 +1,124 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.message2;
+
+import java.io.IOException;
+import java.text.AttributedCharacterIterator;
+
+import com.ibm.icu.text.ConstrainedFieldPosition;
+import com.ibm.icu.text.FormattedValue;
+
+/**
+ * Very-very rough implementation of FormattedValue, packaging a string.
+ * Expect it to change.
+ *
+ * @internal ICU 72 technology preview
+ * @deprecated This API is for ICU internal use only.
+ */
+@Deprecated
+public class PlainStringFormattedValue implements FormattedValue {
```

Comment:
Can we keep this internal / only-package-visible?

Reply:
I use it in the unit tests when I implement some custom functions.

But I can make a copy of it in tests, and make them both package visible?

Reply:
Internal, deprecated, but not tech preview.
visible for testing, maybe.
Search examples.

Reply:
Used `@internal ICU 72 technology preview. Visible For Testing.` as a compromise.

`PluralRules` use `@internal Visible For Testing`, and it is the only class doing that, so I used it as a model.

Not sure it is 100% ok.
But it looks like what's in @internal is not that strictly defined.
I though it is good to keep the "tech preview" part (for example to grep).
There are all kind of "unstructured" things, for example:
```
@internal Used in UnicodeTools
@internal but could become public
@internal, only public for access by callback.
@internal For specific users only until proposed publicly.
```